### PR TITLE
Feature/tailored-meta-data

### PR DESCRIPTION
**Pull request by HugaertsDries**
So tailored meta-files are here! (kind-off)

What are tailored meta-files I hear you ask? Tailored meta-files gives us developers the ability to define meta-data to be extracted, tailored to the semantic-form, without the need to make changes to the service itself. 

I've tried to keep it open for exception to maybe in the future even allow the option to define meta-data to be removed.

I've add an directory `example` containing how this should be configured (within the root config directory of this service).

### CODE REVIEW

File: example/tailored/meta/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const example = require('./extractors/example-tailored-meta-data-extractor');
+
+/**
+ * NOTE: order of execution bound to position in the array
+ */
+module.exports = [
+    example
+]
```
Comment (claire-lovisa) [inline example/tailored/meta/index.js:8]: Simple question : is this to be able to use multiple extractor configurations ?

Comment (HugaertsDries) [reply example/tailored/meta/index.js:8]: It exports an array, so yes! You can dump "as many" as you want.

Comment (HugaertsDries) [reply example/tailored/meta/index.js:8]: These extractors will be run once on creation of a new semantic form and their output dumped into a dedicated tailored additions ttl! :)

File: app.js [hunk 2]
```diff
@@ -261,7 +262,22 @@ app.get('/meta/extract', async function(req, res, next) {
       return next(e);
     }
   }
-  return res.status(403).set('content-type', 'application/n-triples').send();
+  return res.status(403).set('content-type', 'plain/text').send();
+});
+
+app.get('/meta/tailored/extract', async function(req, res, next) {
+  if (DEV_ENV) {
```
Comment (cecemel) [inline app.js:269]: why this DEV_EV, in prod i would just not expose it

File: lib/entities/semantic-form-meta.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  constructor(sources) {
+    this.sources = sources;
+
+    /* Initialize store with meta TTL */
+    this.store = new Graph();
+    if (this.global) {
+      parse(this.global.content, this.store, {graph: GRAPHS.content});
+    }
+    if (this.tailored.additions) {
```
Comment (cecemel) [inline lib/entities/semantic-form-meta.js:25]: (not needed for this PR) but a removal would just run before the additions, and remove the triples in the store?

**Review by cecemel — APPROVED** [review]

**Pull request merged**